src/status_im2/contexts/wallet/select_address_to_watch/view.cljs

```diff
@@ -37,12 +39,17 @@
           :container-style {:margin-right 12
                             :flex         1}
           :weight          :monospace
-          :on-change       #(reset! input-value %)
+          :on-change       #(reset! input-value (.. % -nativeEvent -text))
           :default-value   @input-value}]
         [quo/button
          {:icon-only? true
           :type       :outline} :i/scan]]
-       [rn/view {:style (style/button-container bottom)}
-        [quo/text "[WIP] Bottom Actions"]]])))
+       [quo/button
+        {:customization-color customization-color
+         :disabled?           (clojure.string/blank? @input-value)
+         :on-press            #(re-frame/dispatch [:navigate-to
+                                                   :address-to-watch-edit
+                                                   {:address @input-value}])
+         :container-style     (style/button-container bottom)} (i18n/label :t/continue)]])))
```

Comment:
I think it's preferred if `(i18n/label :t/continue)` is on a newline

Reply:
done